client/src/components/Console.tsx

```diff
@@ -11,6 +13,48 @@ export default function Console() {
     const [groupKey, setGroupKey] = useState("1");
     const [selectedView, setSelectedView] = useState(0);
     const [selectedDocuments, setselectedDocuments] = useState<string[]>(['doc1']);
+    const [documents, setDocuments] = useState<KxDocument[]>([]);
```

Comment:
<img width="1440" alt="Screen Shot 2024-11-12 at 17 22 34" src="https://github.com/user-attachments/assets/275be371-c242-48a9-99ea-c845802c8ca4">
I see this error on the console